EIPS/eip-1753.md

```diff
@@ -43,39 +43,39 @@ This EIP seeks to define a standard that will allow for the granting and/or mana
 Returns the name of the permit - e.g. `"MyPermit"`.
 
 ``` js
```

Comment:
@bitcoinbrisbane I arrived too late but the code looks nicer if you use
```
```solidity
```

instead of 
```
```js
```